### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by bobmcwhirter**
- Add 'fractions' concept
- Add HOWTO for autodetecting and expliciting fractions.

-----

### CODE REVIEW

File: docs/concepts/fractions.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+[#fractions]
+= Fractions
+
+WildFly Swarm is defined by an unbounded set of capabilities. 
+Each chunk of functionality is called a _fraction_.
+Some fractions directly align with commonly-used APIs, such as _JAX-RS_ or _CDI_.
+Other fractions exist to provided higher-order capabilities, such as integration with RHSSO (KeyCloak).
```
Comment (Ladicek) [inline docs/concepts/fractions.adoc]: "to provided" -> "to provide"

"higher-order" sounds maybe too fancy?

"KeyCloak" -> "Keycloak" (not sure about RHSSO, maybe we should use the full form?)

File: docs/concepts/fractions.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
+[#fractions]
+= Fractions
+
+WildFly Swarm is defined by an unbounded set of capabilities.
```
Comment (tradej) [inline docs/concepts/fractions.adoc]: Can you please explain this sentence? Does it mean that it does whatever you want it to?

File: docs/concepts/fractions.adoc (hunk 1 above)
Comment (tradej) [reply docs/concepts/fractions.adoc]: Addressed in bobmcwhirter/wildfly-swarm#2.

**Review by tradej — CHANGES_REQUESTED** [review]
Nice job! For small details, see inline comments.

Please, also merge bobmcwhirter/wildfly-swarm#2.

**Comment by tradej** [comment]
Please do not merge until bobmcwhirter/wildfly-swarm#2 is re-merged--I have forgotten to push all my changes there.

**Review by tradej — APPROVED** [review]

**Comment by tradej** [comment]
Just in case: From my point of view, this can be merged.

**Pull request merged**